### ISSUE TITLE
508-defect-2: File a supplemental claim `h1` should receive focus after subtask is completed 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#54624]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

https://user-images.githubusercontent.com/14154792/223211052-408937f4-c873-45fe-8532-8224576b49af.mov

Upon completing the introduction subtask, the user is navigating to the supplemental claim page where the focus becomes lost in the breadcrumbs.

### Reproduction steps

1. [Complete the subtask flow](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/introduction)
2. Tab, and confirm the focus halo appears in the breadcrumb

### Proposed solution or next steps

- [ ] Focus is sent to the `h1` of the introduction page upon completing the subtask flow

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by saderagsdale** [comment]
@joshkimux can you review this one?

**Comment by joshkimux** [comment]
<img width="701" alt="Screenshot 2023-03-15 at 12 25 59 PM" src="https://user-images.githubusercontent.com/14154792/225375546-29e8fca7-12ac-4bad-a59b-4bbad1689206.png">

PERFECT @rocio-desantiago 💯
